### DISCUSSION TITLE
请问 xmake 在 add_requires 第三方包时，可以不自动拉取 submodule 吗？

### DISCUSSION THREAD

**Discussion by ivanallen** [xmake-io/xmake#6024] [Q&A] [answered]
因为有时候第三方包的 submodule 没有的话，也可以编译，只是不需要这个功能。

**Comment by star-hengxing** [comment, marked as answer]
我打包有留意这个的，大部分都会加上 `add_urls("xxx.git", {submodules = false})`
如果一些老包或者包的新版本不需要 submodule ，可以来个 pr 补上。
